#161677517 Refactor user admin seed data to add isConfirmed user column and set default value

ucheg6 opened the pull request:
[Finishes #161677517]
#### What does this PR do?
- Refactor user admin seed data to add isConfirmed user column and set default value
#### Description of Task to be completed?
- Add the user column isConfirmed to the user admin seed data and set the default value to true

#### How should this be manually tested?
- git clone ` https://github.com/andela/haven-ah-backend.git `
- run npm install to install necessary npm packages
- create dot-env file and setup necessary variables for database configuration
-  run ` createdb haven-ah `to create the database
- run ` sequelize:migrate ` to migrate model
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161677517](https://www.pivotaltracker.com/story/show/161677517)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

victor-nweze-deel commented on server/seeders/20181026125921-user-admin.js:
```diff
@@ -15,6 +15,7 @@ module.exports = {
       email: process.env.ADMIN_EMAIL,
       password: hashedPassword,
       role: 'admin',
+      isConfirmed: 'false',
```
admin should be confirmed by default

ucheg6 replied:
Thanks Victor. I just changed it

seyi-adeleke reviewed (changes requested):
You didn't install or remove any packages so why is there a diff in your package.lock?

ucheg6 commented:
@seyi-adeleke I guess is when I tried to resolve a merge conflict

ucheg6 committed:
Refactor user admin seed data to add isConfirmed user column and set default value

- Add user column isConfirmed to seed data for User Admin

[Finishes #161677517]

seyi-adeleke reviewed (commented):
@ucheg6 Please ensure this works correctly on prod. You should probably sync with @ProfJigsaw
